dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -351,29 +359,19 @@ class AsyncRequestHandler : public UnaryCallback<bool>
             setDone(done_msg);
     }
 
-    bool sendPackets(std::string & err_info)
+    bool sendPackets()
     {
         // note: no exception should be thrown rudely, since it's called by a GRPC poller.
         for (size_t i = 0; i < read_packet_index; ++i)
         {
             auto & packet = packets[i];
-            // We shouldn't throw error directly, since the caller works in a standalone thread.
-            try
```

Comment:
I seems `pushPacket` can throw exception except recomputeTrackedMem such as `fiu_do_on(FailPoints::random_receiver_async_msg_push_failure_failpoint, push_succeed = false;);`

Reply:
This failpoint just set the return value(`push_succeed`) to false, it does not throw exception.